src/books/admin.py

```
@@ -11,3 +11,8 @@ class BookAdmin(admin.ModelAdmin):
 @admin.register(Tag)
 class TagAdmin(admin.ModelAdmin):
     list_display = ('name',)
+
+
+@admin.register(Product)
```

Ruff (0.8.0)

Undefined name `Product`

(F821)